eo-runtime/src/test/eo/org/eolang/runtime-tests.eo

```diff
@@ -415,3 +415,12 @@
           ^.x
     $.equal-to
       "[Hello, Jeff]"
+
+[] > check-triple-quotes
+  assert-that > @
+    """
+    Hello
+
+    Hello
+    """
+    $.equal-to "Hello\n\nHello"
```

Comment:
Does `\n` always mean lineseparator in eo?

Reply:
@levBagryansky Both `\n` and `\r\n` are [valid](https://github.com/objectionary/eo/blob/master/eo-parser/src/main/antlr4/org/eolang/parser/Program.g4#L264-L266) linebreak